go-controller/hack/update-codegen.sh

```diff
@@ -103,3 +103,5 @@ echo "Copying adminpolicybasedexternalroutes CRD"
 cp _output/crds/k8s.ovn.org_adminpolicybasedexternalroutes.yaml ../dist/templates/k8s.ovn.org_adminpolicybasedexternalroutes.yaml.j2
 echo "Copying egressService CRD"
 cp _output/crds/k8s.ovn.org_egressservices.yaml ../dist/templates/k8s.ovn.org_egressservices.yaml.j2
+echo "Copying IPAMClaim CRD"
+curl -sSL https://raw.githubusercontent.com/maiqueb/persistentips/main/artifacts/k8s.cni.cncf.io_ipamclaims.yaml -o ../dist/templates/k8s.cni.cncf.io_ipamclaims.yaml
```

Comment:
Isn't the plumbing group project ready ? is this is the case add a TODO or the like.

Reply:
Done.